maintainers/maintainer-list.nix

```diff
@@ -7144,6 +7144,12 @@
     githubId = 15128988;
     name = "Maksim Dzabraev";
   };
+  dzervas = {
```

Comment:
Ping @dzervas, just to make sure you agree to being added as maintainer here?

Reply:
hey sorry for being late!
yes I'm more than happy to be a maintainer as well!

@randoneering thanks for your work!